src/server/DedicatedServer.cpp

```diff
@@ -342,20 +321,16 @@ void DedicatedServer::printAllGames(std::ostream& stream) const
 }
 
 // special packet processing
-void DedicatedServer::processBlobbyServerPresent( const packet_ptr& packet)
+void DedicatedServer::processBlobbyServerPresent( PlayerID source, RakNet::BitStream& stream )
 {
-	RakNet::BitStream stream(packet->data, packet->length, false);
-
 	// If the client knows nothing about versioning, the version is 0.0
 	int major = 0;
 	int minor = 0;
 	bool wrongPackageSize = true;
 
-	// current client has bitSize 72
-
-	if( stream.GetNumberOfBitsUsed() == 72)
+	// current client has bitSize 64
+	if( stream.GetNumberOfBitsUsed() == 64)
```

Comment:
because further up the call chain,
```
RakNet::BitStream stream(packet->data, packet->length, false);
stream.IgnoreBytes(1);	// ignore the message id
processSingleMessage(static_cast<MessageType>(packet_id), packet->playerId, stream);
```
the message id byte is ignored now, but now that I look at this again it seems wrong. I guess I was thinking of number of bits left.